tests/app/views/test_drafts.py

```diff
@@ -884,8 +897,8 @@ def test_copy_draft(self):
             content_type='application/json')
 
         data = json.loads(res.get_data())
-        assert_equal(res.status_code, 201)
-        assert_equal(data['services']['lot'], 'SCS')
+        assert_equal(res.status_code, 201, res.get_data())
```

Comment:
That's exactly what it does: it'll show the error response instead of the `400 != 201` that we usually get on failures. I changed it while looking into test errors and figured I'd commit it since it is a bit more helpful.